modules/react/popup/lib/PopupCard.tsx

```diff
@@ -82,9 +71,10 @@ export const PopupCard = createComponent('div')({
       <StyledPopupCard
         as={Element}
         transformOrigin={transformOrigin}
-        depth={depth}
-        width={width}
-        padding={padding}
+        position="relative"
+        padding="l"
+        depth={2}
+        maxWidth={`calc(100vw - ${space.l})`}
```

Comment:
ah i see, you moved them out of emotion and on to the element, and then anything they pass as props will override